recipes/traits/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: traits
+  version: {{ version }}
+
+source:
+  fn: traits-4.5.0.tar.gz
+  url: https://pypi.python.org/packages/source/t/traits/traits-4.5.0.tar.gz
+  md5: 3ad558eebaedc63c29c80183c0371d2f
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  run:
+    - python
+  build:
+    - python
+    - setuptools
```

Comment:
Could we please swap the order here? Have `build` requirements before `run` requirements. Also, a space between the two subsections would be nice.